edx_proctoring/static/proctoring/js/exam_action_handler.js

```diff
@@ -48,6 +48,20 @@ edx = edx || {};
             });
     }
 
+    function logError(error, url) {
```

Comment:
A thought on making this call here. It might make more sense to have the special-exams library handle it. For example, when ping is called right now we return the worker promise to the react app and call that actionURL. In that case we'd end up calling the endpoint twice now, which isn't a huge deal but overall think we should do things on the react side if at all possible.

Reply:
Ohhh interesting - okay, let me look into that now.

Reply:
I guess my suggestion is to simply return the promise for each of the functions exposed on this.

Reply:
I think I may have misunderstood this a bit. My general suggestion still stands, I was thinking this code was called within the worker for some reason. All of this is already implemented separately on the react side, we mean that need to change this file at all so long as we don't break anything.

see: https://github.com/edx/frontend-lib-special-exams/blob/addb8f8fcba1dedbe582ee9d4f3c2a2a52d6950a/src/data/messages/handlers.js